src/AspNetCoreModuleV2/CommonLib/HandleWrapper.h

```diff
@@ -10,37 +10,41 @@ struct InvalidHandleTraits
 {
     using HandleType = HANDLE;
     static const HANDLE DefaultHandle;
-    static void Close(HANDLE handle) { CloseHandle(handle); }
+    static void Close(HANDLE handle) noexcept { CloseHandle(handle); }
 };
 
 struct NullHandleTraits
 {
     using HandleType = HANDLE;
-    static constexpr HANDLE DefaultHandle = NULL;
-    static void Close(HANDLE handle) { CloseHandle(handle); }
+    static constexpr HANDLE DefaultHandle = nullptr;
+    static void Close(HANDLE handle) noexcept { CloseHandle(handle); }
 };
 
 struct FindFileHandleTraits
 {
     using HandleType = HANDLE;
     static const HANDLE DefaultHandle;
-    static void Close(HANDLE handle) { FindClose(handle); }
+    static void Close(HANDLE handle) noexcept { FindClose(handle); }
 };
 
 struct ModuleHandleTraits
 {
     using HandleType = HMODULE;
-    static constexpr HMODULE DefaultHandle = NULL;
-    static void Close(HMODULE handle) { FreeModule(handle); }
+    static constexpr HMODULE DefaultHandle = nullptr;
+    static void Close(HMODULE handle) noexcept { FreeModule(handle); }
 };
 
+// Code analysis doesn't like nullptr usages via traits
+#pragma warning( push )
+#pragma warning ( disable : 26477 ) // disable  Use 'nullptr' rather than 0 or NULL (es.47).
```

Comment:
Can we just swap to NULL for these values?

Reply:
They were NULL and were changed to nullptr but it didn't help.

Reply:
I see.